pulp_container/app/registry_api.py

```diff
@@ -1009,6 +980,8 @@ def put(self, request, path, pk=None):
         else:
             add_content_units = [str(manifest.pk)]
             remove_content_units = []
+        # TODO: Is this necessary, or does the the manifest pull it's blobs automatically?
```

Comment:
if i get your question right - it is necessary,  you need to add these blob pks as part of add_remove_task, so you add tag, manifest, blobs in one go